lib/virtual-fs/src/mem_fs/filesystem.rs

```diff
@@ -1520,11 +1520,13 @@ mod test_filesystem {
                     modified,
                     len: 0
                 }) if
-                    accessed == foo_metadata.accessed &&
-                    created == foo_metadata.created &&
+                    accessed <= foo_metadata.accessed &&
+                    created <= foo_metadata.created &&
```

Comment:
This test seemed to be spec'd wrong. Since the root directory was created before `foo`, `accessed` and `created` should be less than or equal to foo's.